cut-n-paste/toolbar-editor/egg-toolbars-model.c

```diff
@@ -597,7 +597,11 @@ egg_toolbars_model_load_toolbars (EggToolbarsModel *model,
     return FALSE;
   }
   root = xmlDocGetRootElement (doc);
-
+  if (root == NULL){
```

Comment:
code formatting: blank space is missing, between the closing parenthesis and the opening curly bracket